NomadAudio/src/AudioDeviceManager.cpp

```
@@ -1,11 +1,41 @@
 #include <chrono>
 
+#ifdef _WIN32
+#include <Windows.h>
+#include <TlHelp32.h>
+#endif
+
 namespace Nomad {
 namespace Audio {
 
+namespace {
+    bool isSoundIDRunning() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)